cdap-distributions/bin/prep_and_stage_bundles.sh

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env bash
+#
+# Copyright © 2014-2015 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+######################################################################################
+# Create RPM and Debian bundles and upload them directly to docs1
+# This script expects to receive 3 parameters in this order:
+#   $1 USER                     REMOTE_USER
+#   $2 remote host              REMOTE_HOST
+#   $3 target directory         REMOTE_INCOMING_DIR
+######################################################################################
+DEBUG=${DEBUG:-no}              ### set to yes for debugging
+RUN_DATE=`date '+%Y%m%d_%R'`
+SCRIPT=`basename ${BASH_SOURCE[0]}`                     ### Set Script Name variable
+BUILD_PACKAGE='cdap'                                   ## e.g. cdap ## a.k.a. BUILD_PACKAGE
+VERSION=$(cdap-distributions/target/stage-packaging/opt/cdap/distributions/VERSION)
+REMOTE_USER=${1}                                        ### remote user
+REMOTE_HOST=${2:-127.0.0.1}                             ### remote host
+REMOTE_INCOMING_DIR=${3}                                ### target directory on remote host
+BUILD_PACKAGE=${BUILD_PACKAGE:-cdap}
+REMOTE_BASE_DIR="incoming"
+USER=${USER:-bamboo}
```

Comment:
Unused, overrides a built-in shell variable, *and* defaults to `bamboo` when there should be no mention of Cask-infrastructure-specific items.